inc/saiport.h

```diff
@@ -1086,6 +1086,17 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PORT_POOL_LIST,
 
+    /**
+     * @brief Pause port egress
+     *
+     * Enable/Disable egress pause of a port.
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_EGRESS_PAUSE_ENABLE,
```

Comment:
I think we can achieve the same functionality by
SAI_PORT_ATTR_QOS_SCHEDULER_PROFILE_ID
And using a profile with SAI_SCHEDULER_ATTR_MAX_BANDWIDTH_RATE of 0
Is this different ?

Reply:
This STOP_PORT_MAX_RATE approach does not work on brcm platform.

Reply:
Perhaps this should be a vendor specific attribute then ?
The scheduler profile approach is something all vendors agreed upon long time ago, having 2 ways to control the same functionality isn't the best approach IMO

Reply:
scheduler profile approach is a mean rather than the purpose, usually SAI attribute defines the purpose not the means.

Reply:
Request name change to TX_ENABLE or STOP_TX.
Comments should specify that this should not drop frames, just stop transmit and buffer.